### DISCUSSION TITLE
Migration from Qmail

### DISCUSSION THREAD

**Discussion by wencywww** [docker-mailserver/docker-mailserver#2645] [Q&A] [answered]
Is it possible and how do i migrate my Email accounts (together with all their mails) from Qmail to Docker-Mailserver? 
The Qmail installation uses vpopmail + dovecot and messages are stored in **maildir** format.

**Comment by casperklein** [comment, marked as answer]
That should be no problem, I did the same some years ago. DMS also uses the maildir format. Just copy over the files and you should be fine.

Edit: You might also take a look at https://github.com/imapsync/imapsync

**Reply by wencywww** [reply]
Sounds great! Thanks.
